playwright/tests/accounts.spec.ts

```diff
@@ -0,0 +1,123 @@
+import { Page, expect, test } from '@playwright/test'
+import { RuntimeAccount } from '../../src/oasis-nexus/api'
+
+async function setup(page: Page) {
+  await page.route(
+    'https://api.coingecko.com/api/v3/simple/price?ids=oasis-network&vs_currencies=usd',
+    route => {
+      // Don't respond
+    },
+  )
+  await page.route('**/v1/', route => {
+    // Don't respond
+  })
+  await page.route('**/v1/sapphire/status', route => {
+    // Don't respond
+  })
+  await page.route(
+    '**/v1/sapphire/transactions?limit=10&offset=0&rel=oasis1qq2v39p9fqk997vk6742axrzqyu9v2ncyuqt8uek',
+    route => {
+      route.fulfill({
+        body: JSON.stringify({
+          is_total_count_clipped: false,
+          total_count: 0,
+          transactions: [],
+        }),
+      })
+    },
+  )
+  await page.route('**/v1/sapphire/accounts/oasis1qq2v39p9fqk997vk6742axrzqyu9v2ncyuqt8uek', route => {
+    route.fulfill({
+      body: JSON.stringify({
+        address: 'oasis1qq2v39p9fqk997vk6742axrzqyu9v2ncyuqt8uek',
+        balances: [
+          {
+            balance: '116404417198000000000',
+            token_decimals: 18,
+            token_symbol: 'TEST',
+          },
+        ],
+        evm_balances: [],
+        stats: {
+          num_txns: 0,
+          total_received: '0',
+          total_sent: '0',
+        },
+      } satisfies Partial<RuntimeAccount>),
+    })
+  })
+  await page.route('**/v1/sapphire/events?rel=oasis1qq2v39p9fqk997vk6742axrzqyu9v2ncyuqt8uek', route => {
+    route.fulfill({
+      body: JSON.stringify({
+        is_total_count_clipped: false,
+        total_count: 3,
+        events: [
```

Comment:
I would add a type like `} satisfies Partial<RuntimeAccount>` above, in case any fields get renamed in the future